packages/web-components/src/components/carousel/carousel.ts

```diff
@@ -107,10 +119,57 @@ class DDSCarousel extends LitElement {
         // @ts-ignore
         this._observerResizeContainer = new ResizeObserver(this._observeResizeContainer);
         this._observerResizeContainer.observe(contentsNode);
+        containerNode?.addEventListener('scroll', this._handleScrollFocus);
```

Comment:
Could you attach this event listener with `lit-html` template syntax? In this way we don't need to explicitly remove the event listener.